src/styles/button.css

```diff
@@ -1 +1,60 @@
 /* button */
+
+.ui.button {
+  background-color: var(--primary-pms-362_900);
+  color: var(--shade-white);
+  font-family: var(--gotham);
+  border-radius: var(--border-radius-3);
```

Comment:
I think we need to decrease the padding of the button.

```
padding: 0.625rem;
```

Reply:
Sounds good!